projects/redemption_configs/configs_specs/configs/specs/config_spec.hpp

```diff
@@ -511,6 +511,14 @@ void config_spec_definition(Writer && W)
         W.member(no_ini_no_gui, rdp_connpolicy, L, type_<RdpModeConsole>(), "mode_console", set(RdpModeConsole::allow), desc{"Console mode management for targets on Windows Server 2003 (requested with /console or /admin mstsc option)"});
 
         W.member(hidden_in_gui, rdp_connpolicy | advanced_in_connpolicy, L, type_<bool>(), "auto_reconnection_on_losing_target_link", set(false));
+
+        W.member(hidden_in_gui, rdp_connpolicy, L, type_<bool>(), "forward_client_build_number",
+            desc{
+                "Forward the build number advertised by the client to the server. "
+                "If forwarding is disabled a default (static) build number will be sent to the server.\n"
+                "Enabling this setting may be required for correct support of some smartcard models."
+            },
+            set(true));
```

Comment:
I suggest to set advanced_in_connpolicy.
And remove the last comment line "Enabling this ..." since this is enabled by default.